language/move-prover/tests/sources/functional/loop_invariant_invalid.move

```diff
@@ -0,0 +1,29 @@
+module 0x42::InvalidLoopInvariants {
```

Comment:
Please add couple more test cases to show that it is not allowed to have assume/asserts before the invariants? Also, I was wondering if it is possible to have only assume/assert statements in the loop header without any invariants?

Reply:
> if it is possible to have only assume/assert statements in the loop header without any invariants?

Yes, this is allowed.

Reply:
In that case, perhaps add test case/s to illustrate that assume/assert can occur by themselves in the loop header block.